Sources/Hypertext.swift

```diff
@@ -6,18 +6,23 @@
 //  Copyright © 2016 Sahand Nayebaziz. All rights reserved.
 //
 
-public protocol Renderable {
-    func render() -> String
-    func render(startingWithSpacesCount: Int) -> String
+public protocol Renderable: CustomStringConvertible {
```

Comment:
Conformance and sharing with `CustomStringConvertible` was something that I tried, but I ended up leaving it out because I could not find a way to do it that was nice. I'm happy to support this, and I like where your head's at